src/__swaps__/screens/Swap/hooks/useEstimatedGasFee.ts

```diff
@@ -19,9 +19,22 @@ function safeBigInt(value: string) {
   }
 }
 
+const isFeeNaN = (value: string | undefined) => isNaN(Number(value)) || !value;
```

Comment:
I think better to be 'value !== undefined' instead of '!value' in case value is 0. Not sure if that case realistic tho